test/mode_handler.test.js

```diff
@@ -18,7 +18,9 @@ test('returned API', t => {
   t.equal(typeof mh.mouseout, 'function', 'exposes mouseout');
   t.equal(typeof mh.keydown, 'function', 'exposes keydown');
   t.equal(typeof mh.keyup, 'function', 'exposes keyup');
-  t.equal(Object.keys(mh).length, 11, 'no unexpected properties');
+  t.equal(typeof mh.click, 'function', 'exposes combineFeatures');
```

Comment:
Also, should I add back the no unexpected properties test? I don't know why I removed that...